### PR TITLE
Copy ns() definition from av1 spec and add serialization algorithm

### PR DESCRIPTION
**Pull request by murillo128**
as per #157

### CODE REVIEW

File: av1-rtp-spec.md [hunk 1]
```diff
@@ -785,6 +799,28 @@ ns(n) {
...
+  x = n
+  while (x != 0) {
+    x = x >> 1
+    w++
+  }
+  m = (1 << w) - n
+  if (v< m)
+    write(w - 1, val);
+  else
+    write(w, v + m)
```
Comment (DanilChapovalov) [inline av1-rtp-spec.md:817]: same as function write_ns is opposite to ns,
function write is opposite to f.
may be stress it by renaming "write" to "write_f"

File: av1-rtp-spec.md [hunk 2]
```diff
@@ -785,6 +799,28 @@ ns(n) {
...
+write_ns(n,v) {
+  if (n == 1) return
+  w = 0
+  x = n
+  while (x != 0) {
+    x = x >> 1
+    w++
+  }
+  m = (1 << w) - n
+  if (v< m)
```
Comment (DanilChapovalov) [inline av1-rtp-spec.md:814]: nit: add a space between 'v' and '<' for style consistency

File: av1-rtp-spec.md [hunk 3]
```diff
@@ -768,6 +768,20 @@ read_bit() {
 </code></pre>
 
 **ns(n)** - non-symmetric unsigned encoded integer with maximum number of values n (i.e., output in range 0..n-1).
+
+This descriptor is similar to ceiling of the base 2 logarithm of the input n, but reduces wastage incurred when encoding non-power of two value ranges by encoding 1 fewer bits for the lower part of the value range. For example, when n is equal to 5, the encodings are as follows (full binary encodings are also presented for comparison):
```
Comment (Philipel-WebRTC) [inline av1-rtp-spec.md:772]: Nit: for small number we spell them out, please change 2, 1 and 5 to 'two', 'one' and 'five'.

File: av1-rtp-spec.md (hunk 1 above)
Comment (murillo128) [reply av1-rtp-spec.md]: done

File: av1-rtp-spec.md (hunk 3 above)
Comment (murillo128) [reply av1-rtp-spec.md]: I only changed the second 1 and 5, I think `base 2` is more readable than `base two` as a mathematical function

Comment (Philipel-WebRTC) [reply av1-rtp-spec.md]: I think you are right, LGTM!

**Comment by agouaillard-cosmo** [comment]
@murillo128 can you Rebase please?

**Comment by murillo128** [comment]
resolved conflict

**Pull request merged**